packages/orbit-components/src/Toast/ToastRoot.tsx

```diff
@@ -43,13 +25,21 @@ const ToastRoot = ({
   const { startPause, endPause, calculateOffset } = handlers;
 
   return (
-    <StyledWrapper
+    <div
       data-test={dataTest}
       id={wrapperId}
-      $top={topOffset}
-      $bottom={bottomOffset}
-      $left={leftOffset}
-      $right={rightOffset}
+      className={cx(
+        "z-onTop pointer-events-none fixed",
+        "bottom-[var(--toast-root-bottom)] end-[var(--toast-root-end)] start-[var(--toast-root-start)] top-[var(--toast-root-top)]",
+      )}
+      style={
+        {
+          "--toast-root-top": `${topOffset}px`,
```

Comment:
Found some more. Positioning is not correct compared to master. If you'd set the position bottom -> it should be at the bottom of the viewport, now it's not. I guess this is happening because of this: (as it has position fixed and it should be ok) 

<img width="238" alt="Screenshot 2023-11-03 at 11 33 16" src="https://github.com/kiwicom/orbit/assets/14054752/4258db21-248f-4223-8633-4bac2c6a6fb6">